<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="s1">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>10</y>
     <width>31</width>
     <height>551</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="lensFrame">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>200</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="lightFrame">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>130</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cameraFrame">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>270</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lightLabel">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>130</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>光源</string>
   </property>
  </widget>
  <widget class="QLabel" name="cameraLabel">
   <property name="geometry">
    <rect>
     <x>1103</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>相机</string>
   </property>
  </widget>
  <widget class="QLabel" name="lensLabel">
   <property name="geometry">
    <rect>
     <x>1103</x>
     <y>200</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>镜头</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>350</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select/Start</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1031</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="image_box1">
        <property name="text">
         <string>彩色图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="image_box2">
        <property name="text">
         <string>深度图</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
